test/initialization-tests.html

```diff
@@ -82,18 +82,21 @@
         setup(function() {
           return getIframe().then(function(i) {
             iframe = i;
-            getMessageMatching(iframe, function(m) {return m.type === 'error'}).then(function(m) {
+            function isError(m) {return m.type === 'error'}
+            getMessageMatching(iframe, isError).then(function(m) {
               error = m.error;
```

Comment:
Yeah, the iframe sets up a global onerror and postMessages it up. We throw in the teardown if we receive an error.